cmd/get.go

```diff
@@ -20,6 +20,9 @@ var (
 func init() {
 	rootCmd.AddCommand(getCmd)
 
-	getCmd.PersistentFlags().IntP("limit", "l", DEFAULT_GET_LIMIT, "Number of records to return")
+	getCmd.PersistentFlags().IntP("limit", "l", DEFAULT_GET_LIMIT, "Number of records to return. Can be used for paginating results in conjuntion with offset.")
+	getCmd.PersistentFlags().IntP("offset", "o", DEFAULT_GET_OFFSET, "Number of records to skip when retrieved. Can be used for paginating results in conjunction with limit.")
```

Comment:
I like this, and want it to be true, but we need to implement offset into the other entities otherwise this will just be ignored for those and be confusing...

Reply:
Ok, I refactored this so the limit and offset flags are set in the subcommands instead of in the get command. 👍